feat(blob): add abortSignal option to all methods (#650)
vercel/storage: .changeset/cold-seas-hunt.md

---
"@vercel/blob": minor
"vercel-storage-integration-test-suite": patch
---

# Add abortSignal

Adds `abortSignal` option to all methods. This allows users to cancel requests using an AbortController and passing its signal to the operation.

Here's how to use it:

```ts

> const abortController = new AbortController();

Reviewer: @luismeyer I updated the example a bit further

---
"@vercel/blob": minor
"vercel-storage-integration-test-suite": patch
---

> Adds `abortSignal` option to all methods. This allows users to cancel requests using an AbortController and passing its signal to the operation.

Reviewer: @luismeyer Can we also add usage in the release notes? It's always pleasant to have everything in the changelog directly. Something like:

```
Here's how to use it (`put` example):

```

> Adds abortSignal option to all methods. This allows users to cancel requests by using an AbortController and passing it's signal to the operation.

Reviewer: ```suggestion
Adds `abortSignal` option to all methods. This allows users to cancel requests using an AbortController and passing its signal to the operation.
```
